tsconfig.storybook.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Why is this needed?

Reply:
For better eslint integration, I updated eslint so it has a different config for stories with different rules that are more adequate for writing stories, which requires a different tsconfig